src/app/shared-links/Components/LinksTable.tsx

```diff
@@ -114,20 +163,35 @@ export default function LinksTable() {
                   {column.label}
                 </TableCell>
               ))}
+              <TableCell>Actions</TableCell>
             </TableRow>
           </TableHead>
           <TableBody>
             {links
               .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
               .map((row) => (
                 <TableRow hover tabIndex={-1} key={row.id}>
-                  {columns.map((column) => {
+                  {combinedCols.map((column) => {
                     const value = row[column.id];
                     return (
-                      <TableCell key={column.id} align={column.align}>
-                        {column.format
-                          ? column.format(value)
-                          : value?.toString()}
+                      <TableCell
+                        key={column.id + row.active}
+                        align={column.align}
+                      >
+                        {column.id === 'action' ? (
+                          <Tooltip title="Deactivate">
```

Comment:
```suggestion
                                                    <Tooltip title="Deactivate">
                            <span>
                              <Button
                                disabled={!row.active}
                                onClick={() => column.action(row.id)}
                              >
                                {column.label}
                              </Button>
                            </span>
                          </Tooltip>
```

MUI is complaining about the disabled button inside the Tooltip, there is a console error after you disable a link. The Tooltip component needs to listen to the events of its child element. Wrapping the Button inside a span ensures that the Tooltip can still listen to the events even when the Button is disabled.